code/provision/playbooks/kubernetes-cluster.yml

```diff
@@ -38,10 +38,8 @@
   hosts: k8s-master
 
   roles:
-    - role: kubernetes-namespace
+    - role: ingress-controller
       namespace: "{{ test_namespace }}"
```

Comment:
I think we're going to have to rethink this configuration block as `namespace` no longer makes sense. Probably not for this PR as we still need to figure out if we're going to change the url strategy to support multiple instances from a single cluster.